pkg/kubelet/apis/config/v1beta1/defaults.go

```diff
@@ -149,6 +149,9 @@ func SetDefaults_KubeletConfiguration(obj *kubeletconfigv1beta1.KubeletConfigura
 		// Keep the same as default NodeStatusUpdateFrequency
 		obj.CPUManagerReconcilePeriod = metav1.Duration{Duration: 10 * time.Second}
 	}
+	if obj.TopologyManagerPolicy == "" {
+		obj.TopologyManagerPolicy = "preferred"
```

Comment:
shouldn't we have default policy 'none' - meaning, "do not do anything, work as older version of kubelet, even if the feature gate is enabled" ?

Reply:
> @lmdaly:
> The policies are a new concept and give additional functionality to resource allocation for cpu and devices

I agree with @kad that the default value of the config field should preserve current behavior